src/lib.rs

```diff
@@ -63,13 +62,35 @@ pub mod pylib {
     }
 
     #[pyfunction]
-    pub fn refresh_logger() {
-        LOG_RESET_HANDLE.reset();
+    pub fn refresh_logger(py: Python) -> PyResult<()> {
+        use log::LevelFilter;
+        let logging = py.import("logging")?;
+        let python_log_level = logging
+            .getattr("getLogger")?
+            .call0()?
+            .getattr("level")?
+            .extract::<usize>()
+            .unwrap_or(0);
+
+        // https://docs.python.org/3/library/logging.html#logging-levels
+        let level_filter = match python_log_level {
+            0 => LevelFilter::Off,
+            1..=5 => LevelFilter::Trace,
+            6..=10 => LevelFilter::Debug,
+            11..=20 => LevelFilter::Info,
+            21..=30 => LevelFilter::Warn,
+            _ => LevelFilter::Error,
+        };
+
+        let logger = pyo3_log::Logger::default().filter(level_filter);
```

Comment:
We should only create the logger and install it on the first run.
Currently if the user calls `refresh_logger ` in python it panics.

```
import daft
daft.refresh_logger()

# PanicException: called `Result::unwrap()` on an `Err` value: SetLoggerError(())
```